feg/gateway/services/n7_n40_proxy/n7/model_conversion.go

```diff
@@ -30,6 +30,35 @@ import (
 	"magma/lte/cloud/go/protos"
 )
 
+var FailureCodeProtoToN7Map = []n7_sbi.FailureCode{
+	"UNUSED",                 // 0
+	"UNK_RULE_ID",            // 1
+	"RA_GR_ERR",              // 2
+	"SER_ID_ERR",             // 3
+	"NF_MAL",                 // 4
+	"RES_LIM",                // 5
+	"MAX_NR_QoS_FLOW",        // 6
+	"UNKNOWN",                // 7
+	"UNKNOWN",                // 8
+	"MISS_FLOW_INFO",         // 10
+	"RES_ALLO_FAIL",          // 11
+	"UNSUCC_QOS_VAL",         // 12
+	"INCOR_FLOW_INFO",        // 13
+	"PS_TO_CS_HAN",           // 14
+	"APP_ID_ERR",             // 15
+	"NO_QOS_FLOW_BOUND",      // 16
+	"FILTER_RES",             // 17
+	"UNKNOWN",                // 18
+	"MISS_REDI_SER_ADDR",     // 19
+	"CM_END_USER_SER_DENIED", // 20
+	"CM_CREDIT_CON_NOT_APP",  // 21
+	"CM_AUTH_REJ",            // 22
+	"CM_USER_UNK",            // 23
+	"CM_RAT_FAILED",          // 24
+	"UNKNOWN",                // 25
+	"UNKNOWN",                // 26
+}
```

Comment:
Why do you need to define them here instead of defining this on proto file?
Why `PolicyReAuthAnswer_FailureCode_value` is defined in protos and not this one?